postman/assets/dashboards/postman_overview.json

```diff
@@ -0,0 +1,692 @@
+{
```

Comment:
Hi @shashankawasthi88 , can you please change this Dashboard "Timeboard" to a Dashboard "Screenboard" and include Postman's logo? 

Here are some dashboarding best practices with details on using screenboards: https://datadoghq.dev/integrations-core/guidelines/dashboards/.

Thanks!

Reply:
Alex, the latest commit has the Dashboard as "Screnboard" with a Logo. Can you verify?

Reply:
Hi Shashank, it seems your screenboard is not formatted correctly. When I copy the raw JSON from the current PR file and import it into a new screenboard in your account, I cannot get it to display. Can you please try to re-export the JSON in your screenboard and update the file? Thanks!

Reply:
Updated, please check now.